tests/playwright/sanity/context-menu.test.ts

```diff
@@ -0,0 +1,93 @@
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../pages/wp-admin-page';
+import EditorPage from '../pages/editor-page';
+import ContextMenu from '../pages/widgets/context-menu';
+import EditorSelectors from '../selectors/editor-selectors';
+import Style from '../pages/elementor-panel-tabs/style';
+
+test.describe( 'Context menu', () => {
+	test( 'Edit widget test', async ( { page }, testInfo ) => {
+		const editorPage = new EditorPage( page, testInfo );
```

Comment:
maybe extract all the repeating actions to a function?

Reply:
we can extract all these to a Fixture
```
const editorPage = new EditorPage( page, testInfo );
const wpAdminPage = new WpAdminPage( page, testInfo );
const contextMenu = new ContextMenu( page, testInfo );
```

Then we get them from here 
`test( 'Duplicate widget test', async ( { page,  wpAdminPage, EditorPage}, testInfo ) `
But this is not for this PR.